### PR TITLE
fix: Dhis2 10432

### PR DESCRIPTION
**Pull request by lucaCambi77**
[DHIS-10432](https://jira.dhis2.org/browse/DHIS2-10432)

Job done : 

- [x] Adds tracker import strategy in front end to switch between sync and async
- [x] Clean code for tracker import service 
- [x] Adds job id for both strategies to keep track on logs
- [x] Updates tests 

-- Please note import is a reserved word in Java so tracker import packages are renamed to **imports**

### CODE REVIEW

**Comment by stian-sandvold** [comment]
Hey @lucaCambi77 ! Looks great overall, well done!

A couple of comments:
- We usually don't collapse imports for less than 5-8 imports. You can set this up in your IDE to not collapse :) I see there was a couple of collapses that would be great to split these up just to be consistent.
- We don't usually add "Enum" to the names of enums, and the term ImportStrategy is something we have used in other importers as well. I think it would be great to keep the enum name as TrackerImportStrategy, and rather find a different name for what you created as TrackerImportStrategy. How about TrackerImportManager/Handler, or something like this?

File: dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/strategy/tracker/imports/impl/TrackerImportAsyncFalseStrategyImpl.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import org.hisp.dhis.webapi.controller.tracker.TrackerImportReportRequest;
+import org.hisp.dhis.webapi.strategy.tracker.imports.TrackerImportStrategy;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Luca Cambi <luca@dhis2.org>
+ */
+@Component
+@RequiredArgsConstructor
+public class TrackerImportAsyncFalseStrategyImpl implements TrackerImportStrategy
```
Comment (enricocolasante) [inline dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/strategy/tracker/imports/impl/TrackerImportAsyncFalseStrategyImpl.java:43]: TrackerImportSyncStrategyImpl I think is more descriptive and do not rely on implementation detail.
If we change the default parameter to be `sync=true` in the future, we do not need to change this name

File: dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/TrackerImportStrategyEnum.java [hunk 2]
```diff
@@ -30,7 +30,7 @@
 /**
  * @author Morten Olav Hansen <mortenoh@gmail.com>
  */
-public enum TrackerImportStrategy
+public enum TrackerImportStrategyEnum
```
Comment (enricocolasante) [inline dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/TrackerImportStrategyEnum.java:33]: Just saw Stian's comment on that.
Agree with him

**Review by enricocolasante — APPROVED** [review]
Just a couple of naming suggestions

**Review by stian-sandvold — APPROVED** [review]

**Pull request merged**